logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/featureConfig/GetFileSharingStatusUseCaseTest.kt

```diff
@@ -0,0 +1,134 @@
+package com.wire.kalium.logic.feature.featureConfig
+
+import com.wire.kalium.logic.NetworkFailure
+import com.wire.kalium.logic.data.featureConfig.FeatureConfigRepository
+import com.wire.kalium.logic.data.featureConfig.FileSharingModel
+import com.wire.kalium.logic.feature.asset.UpdateAssetMessageDownloadStatusUseCaseTest
+import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.logic.test_util.TestNetworkException
+import com.wire.kalium.network.api.ErrorResponse
+import com.wire.kalium.network.exceptions.KaliumException
+import io.mockative.Mock
+import io.mockative.any
+import io.mockative.classOf
+import io.mockative.eq
+import io.mockative.given
+import io.mockative.mock
+import io.mockative.once
+import io.mockative.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertIs
+
+@ExperimentalCoroutinesApi
+class GetFileSharingStatusUseCaseTest {
+
+    @Test
+    fun givenASuccessfulRepositoryResponse_whenInvokingTheUseCase_thenSuccessResultIsReturned() = runTest {
+        // Given
+        val (arrangement, getFileSharingStatusUseCase) = Arrangement()
+            .withSuccessfulResponse()
+            .arrange()
+
+        // When
+        val actual = getFileSharingStatusUseCase.invoke()
+
+        // Then
+        assertIs<GetFileSharingStatusResult.Success>(actual)
```

Comment:
Small nit but to me `assert(actual is GetFileSharingStatusResult.Success)` reads slightly easier 😅